### PR TITLE
Task 1 : README update for new groupeid and remove inappropriate things && modify LICENSE file

### PR DESCRIPTION
**Pull request by ErnestBidouille**
Changes in README : 
- Changed groupid
- Removed inappropriate things 

Changes in LICENSE
- Added names to LICENSE

Minor tasks :
- All team added Formatter and Sonarlint
- Travis CI added

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,16 +1,15 @@
 = J-Voting
-:groupId: io.github.oliviercailloux.y2018
+:groupId: io.github.j-voting
```
Comment (oliviercailloux) [inline README.adoc]: Ok to remove y2018, but please keep io.github.oliviercailloux. The project will continue and I want it to stay under a gId that I control.

File: LICENSE [hunk 2]
```diff
@@ -1,7 +1,8 @@
 MIT License
 
-Copyright (c) 2018 Olivier Cailloux, Quentin Sauvage, Adèle Thorner, 
-  Laetitia Courgey, Vincent Navales
+Copyright (c) 2019 Théo Chennebault, Louis Ledoux, Emilie Chhean, Anael Dray,
```
Comment (oliviercailloux) [inline LICENSE]: Please write: Copyright (c) 2018 to 2019, Olivier Cailloux, Latitia etc. (2018) ; Théo Chennebault, Louis etc. (2019) (replace etc of course). Please order by alphabetical order of the last name inside each year (I know it was not, that’s a mistake). Please respect the limit set in the text in terms of line length.

**Review by oliviercailloux — APPROVED** [review]
Ok, just minor stuff to change, then you can merge.

Is there a reason not to merge in master?

**Comment by ErnestBidouille** [comment]
@oliviercailloux Yes the reason is to hide the changes of the commits already present on the first PR to master. This is to make it easier for you to read.

File: README.adoc (hunk 1 above)
Comment (ErnestBidouille) [reply README.adoc]: @oliviercailloux Oh right, but yet you indicated to change the groupid, I used the one belonging to us, the one of the organization. Perhaps you should consider clarifying this point for next years.

File: LICENSE (hunk 2 above)
Comment (ErnestBidouille) [reply LICENSE]: @oliviercailloux  Ok, will be done asap.

**Comment by ErnestBidouille** [comment]
@j-voting/melta-jvoting Merge of the previous PR has been completed, so this merge will be done on the master branch.

File: README.adoc (hunk 1 above)
Comment (oliviercailloux) [reply README.adoc]: Good remark, indeed that was not clear.

**Pull request merged**